microsoft/vscode
# Change American time format to ISO time format

This PR fixes #140434

**How to Test**

1. Go to the extensions tab
2. Search for an extension (eg "Prettier")
3. Ensure that the "Released on" and "Last updated" fields follow a YYYY-MM-DD, HH:MM:SS format.
